_data/release.yaml

```diff
@@ -0,0 +1,117 @@
+# This object is used for the new-style /documentation and /download paths
+
+# The version number of the latest release
+latest: 0.13.0
+# All releases (in descending order)
+list:
```

Comment:
Probably we should split this up like I've done with the `_data/documentation`, because it's easier to add a self-contained YAML file when doing a release than to update an existing one.